hasjob/views/listing.py

```diff
@@ -574,8 +574,8 @@ def rejectjob(domain, hashid):
             post.status = POSTSTATUS.REJECTED
             msg = Message(subject="About your job post on Hasjob",
                 recipients=[post.email])
-            msg.body = render_template("reject_email.md", post=post)
-            msg.html = markdown(msg.body)
+            msg.body = render_template('reject_email.md', post=post)
+            msg.html = email_transform(markdown(msg.body), base_url=request.url_root)
```

Comment:
This should be:
```python
msg.html = email_transform(render_template('reject_email.html', post=post), base_url=request.url_root)
msg.body = html2text(msg.html)
```

Also apply email_transform to all other situations in which `msg.html` is assigned a value.